lib/app_funcs.sh

```diff
@@ -17,6 +17,12 @@ function copy_hex() {
   mkdir -p ${build_path}/.mix/archives
   mkdir -p ${build_path}/.hex
 
+  # copying hex is not necessary on the new build system,
+  # which builds in /app (which is the same as $HOME)
+  # https://github.com/HashNuke/heroku-buildpack-elixir/issues/194
+  if [ ${HOME} == ${build_path} ]; then
```

Comment:
With the upcoming build system changes (currently opt in via the `` labs), `$HOME` will no longer be `/app` at build time (but will remain `/app` at runtime):

> As part of this change, HOME will also be changing from /app to a PATH under /tmp (otherwise temporary files/secrets saved to HOME would end up in the slug).
> https://github.com/HashNuke/heroku-buildpack-elixir/issues/194

If this function is running at build time, it needs to use `/app` (or `runtime_erlang_path`) rather than `$HOME` in this comparison.